apps/web/.drizzle/meta/0000_snapshot.json

```diff
@@ -1,5 +1,5 @@
 {
-  "id": "786a1fa5-6c4e-4695-904c-9846b8f80c72",
+  "id": "78f3074f-26ec-4e5d-a5a4-2e88311c2efe",
```

Comment:
???
What the heck happened here?

Reply:
I deleted and regenerated the migrations